package/MDAnalysis/core/groups.py

```diff
@@ -2676,12 +2676,12 @@ def unique(self):
            >>> ag
```

Comment:
Maybe add imports on this docstring too?

Reply:
When I run `make doctest` (from #3925), no failing tests appear related to that docstring. Would it still be better to add imports in this case? The page itself is: https://docs.mdanalysis.org/stable/documentation_pages/core/groups.html#MDAnalysis.core.groups.AtomGroup.unique

Reply:
The original PR states:

> All the code in our docstring examples should be self-contained and run on its own. They should:

> import the correct modules
> demonstrate the desired behaviour
> run correctly

So I do believe we want the imports on every doc example :)

Reply:
Right, thank you for confirming! I added the imports and they actually resolved some more (~19) failing tests.

Reply:
Haha that's great, I'll admit I'm a little unclear on how the doctesting framework works.